embeddings/embedding/auto_flair.py

```diff
@@ -111,3 +151,41 @@ def from_hub(
             raise ValueError(pooling_value_error_msg)
 
         return document_embedding
+
+    @staticmethod
+    def from_file(
+        file_path: pathlib.Path,
+        model_type_reference: str,
+        document_embedding_cls: Union[str, Type[DocumentEmbedding]] = FlairDocumentPoolEmbedding,
+        *args: Any,
+        **kwargs: Any,
+    ) -> FlairEmbedding:
+        """AutoFlairEmbedding that allows to specify pooling for the word embeddings.
+        Available document pooling classes:
+            -'FlairDocumentPoolEmbedding'
+            -'FlairTransformerDocumentEmbedding'
+            -'FlairDocumentRNNEmbeddings'
+            -'FlairDocumentCNNEmbeddings'
+        """
+        pooling_value_error_msg: Final = (
+            f"{document_embedding_cls} not recognized as valid document pooling class."
+        )
+
+        if isinstance(document_embedding_cls, str):
+            try:
+                document_embedding_cls = getattr(flair_embedding_module, document_embedding_cls)
+            except AttributeError:
+                raise ValueError(pooling_value_error_msg)
+        assert not isinstance(document_embedding_cls, str)
```

Comment:
Maybe this could be just an `else` statement to the initial `if` with a type error being raised here?

Reply:
It cannot be done like that. When `document_embedding_cls` is a string it is replaced with document pooling class and after that, this condition is checked. Perhaps it is not needed but it appears in multiple classes so I will not touch this right now :)